lib/chef/application/client.rb

```diff
@@ -253,6 +253,11 @@ class Chef::Application::Client < Chef::Application
     :description    => "Enable audit-mode with `enabled`. Disable audit-mode with `disabled`. Skip converge and only perform audits with `audit-only`",
     :proc           => lambda { |mo| mo.gsub("-", "_").to_sym }
 
+  option :audit_as_warning,
```

Comment:
@mcquin I am not sure if this option is needed. Ideally in the future we would like to have a way to let certain audit failures not fail the run but I am not sure yet what is the best way to do it. 

Unless there is a need to I will vote to not include this option in this PR.